images/miq-app/Dockerfile

```diff
@@ -1,19 +1,17 @@
-FROM centos:7
+FROM carbonin/manageiq-ruby-container
```

Comment:
I'm not sure how often SCL releases these and it might require coordination with them. In any case, don't think maintaining our own ruby containers is something we want long term.